interfaces/sorting_test.go

```diff
@@ -152,3 +152,42 @@ func (s *SortingSuite) TestBySlotInfo(c *C) {
 		{Snap: &snap.Info{SuggestedName: "name-2"}, Name: "plug-2"},
 	})
 }
+
+func (s *SortingSuite) TestByConnRef(c *C) {
+	list := []*interfaces.ConnRef{
+		interfaces.NewConnRef(
+			&snap.PlugInfo{Snap: &snap.Info{SuggestedName: "name-1"}, Name: "plug-3"},
```

Comment:
Thinking about this, we could use a helper: `newConnRef(a, b, c, d string) interfaces.ConnRef` just to keep the code shorter here.

Reply:
Nice idea, done.